inc/issue.class.php

```diff
@@ -491,11 +491,16 @@ public function getTicketsForDisplay($options) {
    public function rawSearchOptions() {
       $tab = [];
       $hide_technician = false;
+      $hide_technician_group = false;
       if (!Session::isCron()) {
-         $hide_technician = \Entity::getUsedConfig(
-            'anonymize_support_agents',
-            Session::getActiveEntity()
-         );
+         $user = new \User();
```

Comment:
canonical class name is useless here, and makes harder to maintain code on long term.

```suggestion
         $user = new User();
```